packages/server/package.json

```diff
@@ -2,16 +2,56 @@
   "name": "@online-games-shop/server",
   "version": "1.0.0",
   "scripts": {
-    "watch-ts": "tsc -w",
+    "watch": "tsc -w",
     "build": "tsc",
-    "start": "concurrently \"yarn watch-ts\" \"nodemon ./dist/index.js\"",
+    "start": "concurrently \"yarn watch\" \"nodemon ./dist/index.js\"",
```

Comment:
And yeah, you don't really need `watch` command here.
`yarn start` should do everything for development at once.

Just like in the `create-react-app` by default.